src/main/java/Homework2/Owner.java

```diff
@@ -0,0 +1,24 @@
+package Homework2;
+
+public class Owner extends Human implements CanDrink{
+    String placeOfTheWorking = "Owner";
+    Integer realSalary = 10000;
+    String canDrinkTillDrunk = "Until can see other humans";
```

Comment:
access modifier should be private so that you could access those fields via getters/setters
same for all other models